test/e2e/framework/volume_util.go

```diff
@@ -196,6 +196,14 @@ func NewRBDServer(cs clientset.Interface, namespace string) (config VolumeTestCo
 		},
 	}
 	pod, ip = CreateStorageServer(cs, config)
+
+	// Ceph server container needs some time to start. Tests continue working if
+	// this sleep is removed, however kubelet logs (and kubectl describe
+	// <client pod>) would be cluttered with error messages about non-existing
+	// image.
+	Logf("sleeping a bit to give ceph server time to initialize")
+	time.Sleep(20 * time.Second)
```

Comment:
@jsafrane Do you want a const for the sleep time?  Is there a way to make this a loop rather than a hard-coded interval?

Reply:
filled #54282

Reply:
disregard the previous comment, I updated this PR instead with const.